public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -183,15 +185,15 @@ export class GeomapPanel extends Component<Props, State> {
   }
 
   initMapRef = async (div: HTMLDivElement) => {
+    if (!div) {
+      // Do not initialize new map or dispose old map
+      return;
```

Comment:
Does this mean that we need to explicitly handle disposing the map in cases where we have an overlay (like table)? I'm not aware of any other cases where this might be an issue other than potentially enter / exit panel edit mode

Reply:
For table view, the map div doesn't exist, but we will retain the map object for reference. The map will be disposed in componentWillUnmount().

Reply:
ah I see, this logic makes more sense given we weren't checking the div before assigning it to `this.mapDiv` and then were checking it after 🤔 

Nice fix!